AnalyzedMessageRemediation Graph docs

smkjain8 opened the pull request:
This PR is w.r.t to onboarding our AnalyzedMessageRemediation API onto Graph. 

Link to workload PR - [Pull Request 7115491](https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/7115491): Onboard workload

We will complete the workload PR as soon as this PR is completed

Lauragra commented:
@smkjain8 , did you use the files that were autogenerated to author the content in the PR? At a glance, the topics don't conform to the standard topic structure. For details, see the following topics in the Wiki:

https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211778/Graph-doc-stub-generator-tool
https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/344038/Reference-topic-templates

smkjain8 commented:
@Lauragra yes I used the stub generator tool to generate the files. It generated a lot of files for me so I removed the unnecessary files and renamed some of them. But the content in files is created from the tool. I just introduced a few modifications.

Lauragra commented:
> @Lauragra yes I used the stub generator tool to generate the files. It generated a lot of files for me so I removed the unnecessary files and renamed some of them. But the content in files is created from the tool. I just introduced a few modifications.

Ah, okay. Please be careful not to modify the generated topic structure. For example, the H2 section "CSDL representation" is non-standard. Also, in general we don't publish individual topics for enums. Please see the guidance here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211747/Enumerations

Please review the topic templates and update the PR to include the standard required sections. Thanks!

JarbasHorst commented:
Hi @smkjain8, can you please take a look at Laura's [comment](https://github.com/microsoftgraph/microsoft-graph-docs/pull/20286#issuecomment-1446968777) when you will have time?
